File: www/mod_pagespeed/files/patch-common.py
--- tools/gyp/pylib/gyp/common.py.orig	2012-07-11 21:33:14.908184100 -0400
+++ tools/gyp/pylib/gyp/common.py	2012-07-11 21:33:28.963202252 -0400
@@ -353,6 +353,8 @@
     'sunos5': 'solaris',
     'freebsd7': 'freebsd',
     'freebsd8': 'freebsd',
+    'freebsd9': 'freebsd',
+    'freebsd10': 'freebsd',
   }
   flavor = flavors.get(sys.platform, 'linux')
   return params.get('flavor', flavor)

File: audio/timidity/files/patch-common.c
--- common.c.orig	1996-06-01 12:20:19 UTC
+++ common.c
@@ -24,7 +24,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 
-#if defined(SOLARIS) || defined(__WIN32__)
+#if defined(SOLARIS) || defined(__WIN32__) || defined(__FreeBSD__)
 #include <string.h>
 #else
 #include <strings.h>
